packages/kit/src/core/adapt/builder.spec.js

```diff
@@ -1,7 +1,7 @@
-import { rmSync } from 'node:fs';
-import { join } from 'node:path';
+import { existsSync, rmSync } from 'node:fs';
+import { join, dirname } from 'node:path';
 import { fileURLToPath } from 'node:url';
-import { expect, test } from 'vitest';
+import { assert, expect, test } from 'vitest';
 import glob from 'tiny-glob/sync.js';
```

Comment:
how does this work still? is this test actually being run or is it being pulled in as a transitive dependency from some other package?

Reply:
This had me spinning around for a good 15 minutes. Turns out it's a workspace dependency so it's available in all packages in the repository.

Reply:
ah, I wonder if that caused us not to notice some of these issues because it was available there, but not in end user projects

would you mind removing it as a workspace dependency? you can make it a `devDependency` of this project. we've tried to avoid workspace dependencies in favor of per-project dependencies except for stuff like eslint and prettier

Reply:
What package should become a `devDependency`? Lost on context

Reply:
Oop, forgot about this one, tiny-glob should be moved to the kit devDependencies instead of workspace but I think we can just get rid of it altogether.

Reply:
Would you do that so we can finish and merge the PR?